src/main/java/athleticli/ui/Parser.java

```diff
@@ -3,34 +3,47 @@
 import athleticli.commands.ByeCommand;
 import athleticli.commands.Command;
 import athleticli.commands.activity.AddActivityCommand;
+
 import athleticli.commands.diet.AddDietCommand;
 import athleticli.commands.diet.DeleteDietCommand;
 import athleticli.commands.diet.ListDietCommand;
+
 import athleticli.commands.sleep.AddSleepCommand;
 import athleticli.commands.sleep.DeleteSleepCommand;
 import athleticli.commands.sleep.EditSleepCommand;
 import athleticli.commands.sleep.ListSleepCommand;
+
 import athleticli.data.activity.Activity;
 import athleticli.data.activity.Run;
 import athleticli.data.activity.Swim;
+
+import athleticli.commands.diet.EditDietGoalCommand;
+import athleticli.commands.diet.SetDietGoalCommand;
+import athleticli.data.diet.DietGoal;
 import athleticli.data.diet.Diet;
+
 import athleticli.exceptions.AthletiException;
 import athleticli.exceptions.UnknownCommandException;
 
 import java.time.LocalDateTime;
 import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
 
 /**
  * Defines the basic methods for command parser.
  */
 public class Parser {
+    private static final String caloriesMarkerConstant = "calories";
+    private static final String proteinMarkerConstant = "protein";
+    private static final String carbMarkerConstant = "carb";
+    private static final String fatMarketConstant = "fat";
```

Comment:
“Market” seems like a typo.
According to our [coding standard](https://se-education.org/guides/conventions/java/basic.html), constant names must be all uppercase using underscore to separate words.
```suggestion
    private static final String CALORIES_MARKER = "calories";
    private static final String PROTEIN_MARKER = "protein";
    private static final String CARB_MARKER = "carb";
    private static final String FAT_MARKER = "fat";
```